Formula/d/dockerfile-language-server.rb

```diff
@@ -0,0 +1,39 @@
+require "language/node"
+
+class DockerfileLanguageServer < Formula
+  desc "Language server for Dockerfiles powered by Node, TypeScript, and VSCode"
+  homepage "https://github.com/rcjsuen/dockerfile-language-server-nodejs"
```

Comment:
the repo can be renamed cc @gwynne

Reply:
Done - I'd actually done it already, but I fat-fingered when I force-pushed the squash and didn't notice until now 😅